README.md

```diff
@@ -264,6 +266,56 @@ struct User: Unboxable {
 }
 ```
 
+You can also use key paths to directly unbox nested JSON structures. This is useful when you only need to extract a specific object (or objects) out of the JSON body.
+
+```json
+{
+    "company": {
+        "name": "Spotify",
+	},
+	"jobOpenings": [
+		{
+			"title": "Swift Developer",
```

Comment:
Yeah, like I mentioned in my previous comment, tabbing in iA Writer was a bit different.